actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -1,14 +1,17 @@
+require "action_dispatch/system_testing/driver"
+
 module ActionDispatch
   module SystemTesting
-    class Browser # :nodoc:
+    class Browser < Driver # :nodoc:
       def initialize(name, screen_size)
+        super(name)
         @name = name
         @screen_size = screen_size
       end
 
-      def run
+      def use
```

Comment:
Why change `run` to `use`? It's still running the driver/browser and I prefer all the classes (Driver, Browser, and Server) use the same public methods.